ietf/nomcom/views.py

```diff
@@ -1313,14 +1313,17 @@ def eligible(request, year, public=False):
 def public_volunteers(request, year):
     return volunteers(request=request, year=year, public=True)
 
-def private_volunteers(request, year):
-    return volunteers(request=request, year=year, public=False)
+def private_volunteers(request, year, mode):
+    return volunteers(request=request, year=year, public=False, mode="full")
```

Comment:
This seems strange - why is mode an argument to this function if you're always going to ignore what's passed in?

Reply:
copy-editing error.  the `="full"` should be in the parameter declaration.